server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/ScheduledTaskLoggerService.java

```diff
@@ -0,0 +1,224 @@
+import com.oceanbase.odc.service.task.caller.ExecutorIdentifierParser;
+import com.oceanbase.odc.service.task.config.TaskFrameworkEnabledProperties;
+import com.oceanbase.odc.service.task.constants.JobAttributeKeyConstants;
+import com.oceanbase.odc.service.task.executor.logger.LogUtils;
+import com.oceanbase.odc.service.task.model.ExecutorInfo;
+import com.oceanbase.odc.service.task.model.OdcTaskLogLevel;
+import com.oceanbase.odc.service.task.service.TaskFrameworkService;
+import com.oceanbase.odc.service.task.util.JobUtils;
+import com.oceanbase.odc.service.task.util.TaskExecutorClient;
+
+import cn.hutool.core.io.FileUtil;
+import cn.hutool.core.util.ObjectUtil;
+import cn.hutool.core.util.StrUtil;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * @author mayang
+ */
+@Service
+@Slf4j
+public class ScheduledTaskLoggerService {
+
+    private static final String LOG_PATH_PATTERN = "%s/scheduleTask/%s-%s/%s/log.%s";
+
+    private final ScheduleTaskService scheduleTaskService;
+    private final TaskFrameworkEnabledProperties taskFrameworkEnabledProperties;
+    private final RequestDispatcher requestDispatcher;
+    private final TaskDispatchChecker dispatchChecker;
+    private final TaskFrameworkService taskFrameworkService;
+    private final JobDispatchChecker jobDispatchChecker;
+    private final TaskExecutorClient taskExecutorClient;
+
+    private final LoggerProperty loggerProperty;
+    private final CloudObjectStorageService cloudObjectStorageService;
+
+    public ScheduledTaskLoggerService(ScheduleTaskService scheduleTaskService,
+            TaskFrameworkEnabledProperties taskFrameworkEnabledProperties,
+            RequestDispatcher requestDispatcher,
+            TaskDispatchChecker dispatchChecker,
+            TaskFrameworkService taskFrameworkService,
+            JobDispatchChecker jobDispatchChecker,
+            TaskExecutorClient taskExecutorClient,
+            LoggerProperty loggerProperty,
+            CloudObjectStorageService cloudObjectStorageService) {
+        this.scheduleTaskService = scheduleTaskService;
+        this.taskFrameworkEnabledProperties = taskFrameworkEnabledProperties;
+        this.requestDispatcher = requestDispatcher;
+        this.dispatchChecker = dispatchChecker;
+        this.taskFrameworkService = taskFrameworkService;
+        this.jobDispatchChecker = jobDispatchChecker;
+        this.taskExecutorClient = taskExecutorClient;
+        this.loggerProperty = loggerProperty;
+        this.cloudObjectStorageService = cloudObjectStorageService;
+    }
+
+    public String getLog(Long scheduleTaskId, OdcTaskLogLevel level) {
+        try {
+            return LogUtils.getLatestLogContent(getLogFile(scheduleTaskId, level), loggerProperty.getMaxLines(),
+                    loggerProperty.getMaxSize());
+        } catch (Exception e) {
+            log.warn("{}, taskId={}", LogUtils.DEFAULT_LOG_CONTENT, scheduleTaskId, e);
```

Comment:
who is expected to read this log line? 
if not for user, may not use same content as DEfAULT_LOG_CONTENT

Reply:
I have changed it